.stylelintrc.json

```
@@ -0,0 +1,3 @@
+{
+    "extends": "stylelint-config-standard"
+  }
```

```suggestion
}
```
Пожалуйста, пользуйтесь eslint чтобы проверить код перед отправкой на ревью